Mini gravity simulator added

NyuAlpha opened the pull request:
Hello, this is my first pull request in an open source project =)
I have added a small gravity simulator using HTML, CSS, and JavaScript. You can see how particles attract each other, forming clumps that eventually fuse together.
I hope you like it!

NyuAlpha committed:
Mini gravity simulator added

RitamPaul commented:
Congratulations buddy for your 1st contribution in open source. I would like you to implement the followings into your code :-

just increase the particle size so that it would clearly visible and increase the playground area size a little bit

NyuAlpha committed:
Modified simulator size and physics

NyuAlpha commented:
Thank you for the feedback! 
I've made the changes: The particle size and playground area are now bigger. I also adjusted the physics and removed the z coordinate for a smoother appearance.
Looking forward to your thoughts!
